src/components/missingData/MissingData.tsx

```diff
@@ -0,0 +1,29 @@
+import LinkButton from '../linkButton/LinkButton';
+import { LinkType } from '../../../studio/lib/payloads/navigation';
+import styles from './missingData.module.css';
+import textStyles from "src/components/text/text.module.css";
+
+const studioLink = {
+  _key: "go-to-sanity-studio",
+  _type: "link",
+  linkTitle: "Go to Studio",
+  linkType: LinkType.Internal,
+  internalLink: {
+    _ref: "studio",
+  },
+};
+
+export const MissingData = ({ description }: { description: string }) => {
+  return (
+    <div className={styles.wrapper}>
+      <div className={`${textStyles.small} ${styles.container}`}>
```

Comment:
super nitpick, 

but very often when we have classes such as wrapper and container used simultaneously like here (and I'm sure I have done as well in this repo!) it's easy to get confused and mix the two. So I try to name the outside tag either wrapper or the name of the tag like "article" or "section" if they can be those tags. If not I name the ouside wrapper and then the inside content-wrapper more closely related to the component-name itself, but in this case, I'd probably go for something like: "content" 

other times I've done "wrapper" and then "logoSalad"

Reply:
classic problem with semantic class names 😅